lib/engine/game/base.rb

```diff
@@ -414,6 +414,7 @@ def new_stock_round
 
       def new_operating_round(round_num = 1)
         @log << "-- Operating Round #{@turn}.#{round_num} --"
+        @corporations.reject(&:floated?).each { |c| c.push_revenue_history!(nil, round_num) }
```

Comment:
hm i'm not sure i like this

what if revenue history was more detailed

Reply:
Detailed in what way? I'm not sure what you mean. Do you mean if revenue history had turn and round information? Or...?

Reply:
can we get around having to inject this fake data? what if the revenue history had more info so this didn't need to happen

Reply:
It could be a map keyed off of turn and round_num

Reply:
yes, maybe let's try that

Reply:
ok, I'll get to that tomorrow. let me know if there are any big issues :) namely formatting of the cards lol